src/packet/ipv4.rs.in

```diff
@@ -171,7 +177,7 @@ mod checksum_tests {
 
     #[test]
     fn checksum_zeros() {
-        let mut data = vec![0u8; 20];
```

Comment:
It definitely used to be required - otherwise it would infer `usize` instead (or error, I forget).